readme.md

```diff
@@ -114,8 +125,12 @@ Write some basic CSS that demonstrates changing a CSS property when the device w
 
 How is rebase different than a merge?
 
+
 Your Answer:
 ```text
+Rebase reverts all commits since the current branch diverge from <upstream>, and then re-applies them one by one on top of changes from the HEAD of <upstream>.
+
+Merge changes from <branch name> into current branch.
```

Comment:
Close.  I would avoid using "revert" when describing rebase.  Git has a "revert" command, which has a similar, but different purpose.

Please review the answer key in the solution branch.